kernel/packages/shared/catalogs/sagas.ts

```diff
@@ -162,8 +162,15 @@ function* fetchWearablesV2(filters: WearablesRequestFilters) {
   if (filters.ownedByUser) {
     if (WITH_FIXED_COLLECTIONS) {
       const collectionIds = WITH_FIXED_COLLECTIONS.split(',')
-      const wearables = yield call(fetchWearablesByFilters, { collectionIds }, client)
-      result.push(...wearables)
+      const orgCatalyst = 'https://peer.decentraland.org'
```

Comment:
Can we use the Catalyst Client library to give any catalyst from prod instead of hardcoding it here?

Reply:
Done!